.circleci/config.yml

```diff
@@ -428,99 +440,151 @@ workflows:
   version: 2
   build_and_test:
     jobs:
+      - build-local:
+          context: org-global
+          filters:
+            tags:
+              only: /.*/
+
       - setup:
           context: org-global
+          filters:
+            tags:
+              only: /.*/
+            branches:
```

Comment:
I don't think we really need these branch ignores here

Reply:
is this refering to `setup` only?

Reply:
I mean for any branch ignore - apart from the `publish` step

Reply:
done.